azurerm_redis_cache/main.tf

```
@@ -45,3 +45,14 @@ resource "azurerm_redis_cache" "redis_cache" {
     ignore_changes = [redis_configuration[0].rdb_storage_connection_string]
   }
 }
+
+module "lock" {
+  count             = var.lock != null ? 1 : 0
+  source            = "git::git@github.com:pagopa/io-infrastructure-modules-new.git//azurerm_management_lock?ref=v2.1.23"
+  global_prefix     = var.global_prefix
+  environment_short = var.environment_short
+  name              = var.lock.name
```

```suggestion
  name              = local.resource_name
```
Can be useful not specify lock name and get it from resource_name?

---

Not really as the lock name is going to be expanded too:

https://github.com/pagopa/io-infrastructure-modules-new/blob/e66ab13fee34689c1886a43f7761e9fd1020dc21/azurerm_management_lock/vars.tf#L49

locals {
  resource_name = "${var.global_prefix}-${var.environment_short}-lock-${var.name}"
}